### ISSUE TITLE
Create helpdesk policies and framework

### ISSUE THREAD

**Issue by nickolasclarke** [nickolasclarke/QLEAR-Meta#2]
Right now pure living and others send questions, requests, etc related to QLEAR in a variety of formats including 3+ different wechat groups, PM's in wechat, emails to individuals, one-on-one chats, etc. Nearly nothing is documented in a public fashion, tracked, or can be searched. this is not scalable. A framework needs to be developed to create transparency for groups both outside of the QLEAR Development team, as well as QLEAR itself.

## Requirements

* Issue tracking for 3rd party bugs, feature requests, and helpdesk tasks such as making new cities, or other admin only tasks
* accessible to outside parties with little to no cost to QLEAR
* provides way to solicit feedback in both directions
* searchable
* possibly semi-private or private to the general public? 

Providing some sort of limited access to the QLEAR repos could solve all of this without having to introduce new technologies for the _whole_ team (though that may not be true for all the 3rd parties).  If we can clean up some old repos in the GIGA org (?), we can make an issue-only repo that could be the issue tracker for all QLEAR related issues. This could include tagging for use for helpdesk and 3rd party reporting that then could allow us to convert issues into actionable issues with technical requirements without having to look in other repos. There are downsides, however, with a convoluted group of issues that could be hard to parse (tagging can solve lots of that though) as well as difficulty limiting visibility on technical issues that will be assigned to devs. Alternatively, we could keep technical tickets to their respective repos (qlear-web, -app, -api) and use a private or public "issue only" repo for QLEAR called qlear-helpdesk or something. 

Lastly, we could do a hybrid of the two, which would be two issue only repos, one of which would be the master repo for all "technical" issues across the code repos, and a second, Helpdesk only repo. I think this may be the best solution, as this would permit all technical work to be centralized to one repo that people need to look for, while keeping 3rd parties out of the discussion of technical tickets, but also centralizing all communications in a trackable fashion for helpdesk things. Helpdesk tickets, if deemed valid, would spawn new issues in the master technical repo. Issues across repos can be referenced (and even used to close issues automatically with PRs) using the extended syntax of `username/repo/#issue` 

This is convoluted, but enough to remember my thoughts and discuss further.

**Comment by nickolasclarke** [comment]
@stantonwong test if I can mention you here.

**Comment by nickolasclarke** [comment]
@gigarkw you too! since you are actually in the office.

**Comment by gigarkw** [comment]
I’m running away now!!!
